metriq_gym/hardware/clops_benchmark.py

```diff
@@ -369,6 +370,7 @@ def __init__(self,
         circuit_type: str = "twirled",
         batch_size: int = None,
         pipelines: int = 1,
+        job: Job = None,
     ):
         
         """Run CLOPS benchmark through Sampler primitive
```

Comment:
Based on this new file in `hardware/` should we update the README to be up-to-date with how one runs this?

Reply:
Done.